assets/templates/contact-policy.html

```diff
@@ -0,0 +1,180 @@
+    <!-- Font Proxima Nova -->
+    <link rel="stylesheet" href="https://use.typekit.net/fbq8dbp.css" />
+    <!-- Styles -->
+    <link rel="stylesheet" type="text/css" href="./css/styles.css" />
+    <!-- Favicons -->
+    <link
+      rel="apple-touch-icon-precomposed"
+      sizes="144x144"
+      href="https://www.fromdoppler.com/apple-touch-icon-144x144-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon-precomposed"
+      sizes="114x114"
+      href="https://www.fromdoppler.com/apple-touch-icon-114x114-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon-precomposed"
+      sizes="72x72"
+      href="https://www.fromdoppler.com/apple-touch-icon-72x72-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon-precomposed"
+      sizes="57x57"
+      href="https://www.fromdoppler.com/apple-touch-icon-57x57-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon-precomposed"
+      href="https://www.fromdoppler.com/apple-touch-icon-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      href="https://www.fromdoppler.com/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="https://www.fromdoppler.com/favicon.ico"
+    />
+    <link
+      rel="shortcut icon"
+      type="image/x-icon"
+      href="https://www.fromdoppler.com/favicon.ico"
+    />
+  </head>
+
+  <body data-active-menu="34" class="dp-library">
+    <div class="dp-container">
+      <div class="dp-rowflex">
+        <div class="col-sm-12 m-t-24">
+          <header class="m-b-30">
+            <a
+              href="index.html"
+              class="dp-button button-medium secondary-green button--round"
+            >
+              Back
+            </a>
+          </header>
+        </div>
+      </div>
+    </div>
+
+    <header class="hero-banner">
+      <div class="dp-container">
+        <div class="dp-rowflex">
+          <div class="col-sm-12 col-md-12 col-lg-12">
+            <nav class="dp-breadcrumb">
+              <ul>
+                <li>
+                  <a href="#">Panel de Cotrol</a>
+                </li>
+                <li>Política de Contacto</li>
+              </ul>
+            </nav>
+            <h2>Política de Contacto</h2>
+          </div>
+          <div class="col-sm-7">
+            <p>
+              Here you'll find all your Subscription Forms including the ones
+              that are in draft, published or with unpublished changes. You
+              currently have 39 Forms. You can create up to
+              <a href="#">HELP.</a>
+            </p>
+          </div>
+        </div>
+        <span class="arrow"></span>
+      </div>
+    </header>
+    <section class="dp-container">
+      <div class="dp-rowflex">
+        <div class="col-md-6 col-sm-12 m-b-24">
+          <form class="dp-contact-policy-form">
+            <fieldset>
+              <legend>Política de contacto</legend>
+              <ul class="field-group">
```

Comment:
I'm not so sure about rendering all the layout as an ul, this way we do not take advantage of the grid, it shows in smaller resolutions.